src/vcpkg/tools.cpp

```diff
@@ -260,22 +331,24 @@ namespace vcpkg
                                      const ToolData& tool_data)
     {
         auto downloaded_path = fetch_tool(paths, tool_provider.tool_data_name(), tool_data);
-        auto downloaded_version = tool_provider.get_version(paths, downloaded_path).value_or_exit(VCPKG_LINE_INFO);
-        return {std::move(downloaded_path), std::move(downloaded_version)};
+        return {std::move(downloaded_path),
+                ToolVersion::try_parse_numeric(
+                    tool_provider.get_version(paths, downloaded_path).value_or_exit(VCPKG_LINE_INFO))
+                    .value_or_exit(VCPKG_LINE_INFO)};
     }
 
-    static PathAndVersion get_path(const VcpkgPaths& paths, const ToolProvider& tool, bool exact_version = false)
+    static PathAndVersion get_path(const VcpkgPaths& paths, const ToolProvider& tool, RequireExactVersions compare_kind)
```

Comment:
I think leaving this defaulted to "NO" would be fine as well, to reduce verbosity at the callers.